src/main/java/com/spotify/docker/client/messages/Info.java

```diff
@@ -223,20 +231,15 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "Info{" +
-           "containers=" + containers +
-           ", images=" + images +
-           ", storageDriver='" + storageDriver + '\'' +
-           ", executionDriver='" + executionDriver + '\'' +
-           ", kernelVersion='" + kernelVersion + '\'' +
-           ", debug=" + debug +
-           ", fileDescriptors=" + fileDescriptors +
-           ", goroutines=" + goroutines +
-           ", eventsListener=" + eventsListener +
-           ", initPath='" + initPath + '\'' +
-           ", sockets=" + sockets +
-           ", memoryLimit=" + memoryLimit +
-           ", swapLimit=" + swapLimit +
-           '}';
+    return String.format("Info{ containers = %d, images = %d, storageDriver = %s, "
+                         + "driverStatus = %s, cpus = %d, memTotal = %d, name = %s, "
+                         + "executionDriver = %s, kernelVersion = %s, debug = %d, "
+                         + "fileDescriptors = %d, goroutines = %d, eventsListener = %d, "
+                         + "initPath = %s, initSha1 = %s, indexServerAddress = %s, "
+                         + "sockets = %s, memoryLimit = %b, swapLimit = %b",
+                         containers, images, storageDriver, driverStatus, cpus, memTotal, name,
+                         executionDriver, kernelVersion, debug, fileDescriptors, goroutines,
+                         eventsListener, initPath, initSha1, indexServerAddress,
+                         Joiner.on(", ").join(sockets), memoryLimit, swapLimit);
```

Comment:
Haven't used that version so not sure.

On Thu, Mar 19, 2015 at 3:54 PM, Rohan Singh <notifications@github.com>
wrote:

> In src/main/java/com/spotify/docker/client/messages/Info.java
> <https://github.com/spotify/docker-client/pull/143#discussion_r26791278>:
>
> > -           ", eventsListener=" + eventsListener +
> > -           ", initPath='" + initPath + '\'' +
> > -           ", sockets=" + sockets +
> > -           ", memoryLimit=" + memoryLimit +
> > -           ", swapLimit=" + swapLimit +
> > -           '}';
> > +    return String.format("Info{ containers = %d, images = %d, storageDriver = %s, "
> > +                         + "driverStatus = %s, cpus = %d, memTotal = %d, name = %s, "
> > +                         + "executionDriver = %s, kernelVersion = %s, debug = %d, "
> > +                         + "fileDescriptors = %d, goroutines = %d, eventsListener = %d, "
> > +                         + "initPath = %s, initSha1 = %s, indexServerAddress = %s, "
> > +                         + "sockets = %s, memoryLimit = %b, swapLimit = %b",
> > +                         containers, images, storageDriver, driverStatus, cpus, memTotal, name,
> > +                         executionDriver, kernelVersion, debug, fileDescriptors, goroutines,
> > +                         eventsListener, initPath, initSha1, indexServerAddress,
> > +                         Joiner.on(", ").join(sockets), memoryLimit, swapLimit);
>
> Actually I don't see sockets at all with Docker 1.5.0. Do you see it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/143/files#r26791278>.
>



-- 
David Xia
Software Engineer
dxia@spotify.com